test/sslapitest.c

```diff
@@ -8019,8 +8019,8 @@ static int test_pluggable_group(int idx)
     if (!TEST_true(create_ssl_connection(serverssl, clientssl, SSL_ERROR_NONE)))
         goto end;
 
-    if (!TEST_true(!strcmp(group_name, SSL_group_to_name(serverssl,
-                                         SSL_get_shared_group(serverssl, 0)))))
+    if (!TEST_str_eq(group_name, SSL_group_to_name(serverssl,
+                                         SSL_get_shared_group(serverssl, 0))))
```

Comment:
Indentation looks a little odd here

Reply:
Yeah, better split the line after the group_name. However this would mean with proper indent that the second line will be still slightly longer than 80 chars. But I think that should be acceptable.

Reply:
Done -- indeed slightly above the 80... but if you'd think that's OK, indentation certainly is more logical now.